src/mpi/group/grouputil.c

```diff
@@ -89,6 +89,7 @@ int MPIR_Group_init(void)
 int MPIR_Group_finalize(void)
 {
     num_worlds = 0;
+
```

Comment:
Remove blank line. Only updating code format if this is update to code logic. This would reduce potential merge conflict.

Reply:
It's gone.